docs/index.md

```diff
@@ -190,6 +190,16 @@ export YAMLFIX_COMMENTS_REQUIRE_STARTING_SPACE="true"
 
 This option enforces a space between the comment indicator (`#`) and the first character in the comment. It implements the enforcement of the yamllint rule `rules.comments.require-starting-space` - see: https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.comments
 
+### Comments Optional Number Whitelines From  Content
+
+Default: `comments_optional_number_whitelines_from_content: int = 1`<br>
```

Comment:
I feel that the config option is too long, maybe we can name this `comment_whitelines`? I'm always on the side of explicit variable names that let you know exactly what the variable does. In this case as this will be written in the command line, I think it's best to use a shorter form and in the docs and help message explain what does it do. What do you think? 

If you prefer to leave it as is, I'm fine with it too

Reply:
Done in [b8b94cd](https://github.com/lyz-code/yamlfix/pull/208/commits/b8b94cdf7eb7f3d35e983dfb0e29e44f496e346b)

And pluralized in 
1b92ff5 since the other parameters are pluralized (`comments_min_spaces_from_content`, `comments_require_starting_space`)

Reply:
[Question]

Like you, I prefer `comments_whitelines` to the longer `comments_optional_number_whitelines_from_content `.

Are we sure that this is the single best way to name this parameter? (Just asking since once its released, there's no going back)

Reply:
Other that pop into my mind are `section_whitelines`, but I don't know if it's worse xD. Any other ideas?

Reply:
I don't have a better idea.

In search for a better name, I read the [yaml glossary](https://yaml.org/spec/1.2.2/ext/glossary/), and there are no name for a line with no node defined on it.

Reply:
Reworded `index.md` in [649cb41](https://github.com/lyz-code/yamlfix/pull/208/commits/649cb41e2e6efb602ba6f05d08f18412a4ae92ee)